src/styles/moyo__header.scss

```
@@ -0,0 +1,53 @@
+.moyo-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin-bottom: 50px;
+}
+
+.logo-link {
+  width: 40px;
+  height: 40px;
+}
+
+.moyo-nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav-links {
+  margin-left: 20px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+}
+
+.nav-links:first-child {
+  margin-left: 0;
+}
+
+.is-active,
+.nav-links:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  height: 4px;
+  width: 37px;
```

```suggestion
  width: 100%;
```
якщо написати так то при зміні тексту на довший, наприклад, ширина after теж зміниться